builder/oci/driver_oci.go

```diff
@@ -174,6 +174,12 @@ func (d *driverOCI) CreateInstance(ctx context.Context, publicKey string) (strin
 		Metadata:           metadata,
 	}
 
+	if d.cfg.InstanceOptions.AreLegacyImdsEndpointsDisabled != nil {
+		InstanceOptionsDetails := core.InstanceOptions{AreLegacyImdsEndpointsDisabled: d.cfg.InstanceOptions.AreLegacyImdsEndpointsDisabled}
+
+		instanceDetails.InstanceOptions = &InstanceOptionsDetails
```

Comment:
Could be inlined maybe?

Reply:
Sorry @neumayer , I was out last week, so didn't have a chance to get to this until now.  I pulled the instance options inline.  Thank you.

<img width="1342" alt="image" src="https://github.com/user-attachments/assets/93829c5d-a49b-4330-8adb-4ee87bc8b105" />